src/config/chainsRegistry.ts

```diff
@@ -12,19 +15,41 @@ type EVMChainConfig = {
   erc20Contracts: Record<string, ERC20Record>;
 };
 
-export type ChainConfig = EVMChainConfig;
-
-export type ChainRegistry = Record<ChainType, Record<string, ChainConfig>>;
-
 export enum ChainNames {
-  KAVA_EVM = 'Kava EVM',
-  ETH = 'Ethereum',
-  KAVA_EVM_INTERNAL_TESTNET = 'Kava EVM Internal Testnet',
+  KAVA_COSMOS = 'Kava Cosmos',
+
+  KAVA_EVM = 'kavaEVM',
+  ETH = 'eth',
+  KAVA_EVM_INTERNAL_TESTNET = 'kavaEVMInternalTestnet',
```

Comment:
Let's keep these as the "display names" too.

```suggestion
  KAVA_EVM = 'Kava EVM',
  ETH = 'Ethereum',
  KAVA_EVM_INTERNAL_TESTNET = 'Kava EVM Internal Testnet'
```